frontend/src/static/js/components/webstatus-line-chart-tabbed-panel.ts

```diff
@@ -57,6 +57,23 @@ export abstract class WebstatusLineChartTabbedPanel<
     this.resetPointSelectedTask();
   }
 
+  getTabTooltip(tab: string): TemplateResult {
+    if (tab === 'Mobile') {
```

Comment:
This is in the generic tabbed  panel.

You could do this here:

```
  getTabTooltip(tab: string): TemplateResult {
    return html`${tab}`;
  }
```

And then in the WPT implementation panel:

```
  getTabTooltip(tab: string): TemplateResult {
    if (tab === 'Mobile') {
       // Render the tooltip like you have any return
    }

   // Else, use the parent classes tooltip renderer
   return super.getTabTooltip(tab)
  }
```

Reply:
Yes, this is definitely a better implementation - I've updated it!